AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Support/ModelSyncedPayload.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+struct ModelSyncedPayload {
```

Comment:
Since you're emitting this on the Hub, this needs to be a public type. Each member should also be public. It isn't strictly necessary to make the initializer public, but it's good practice.

Reply:
> Since you're emitting this on the Hub, this needs to be a public type. Each member should also be public. It isn't strictly necessary to make the initializer public, but it's good practice.

@DongQuanRui, see this comment, i don't think the members in the other classes are currently public